tests/test_approval_modules.py

```diff
@@ -109,7 +109,7 @@ def fake_read_yaml_files(
             build_dir=str(local_tmp_path),
             build_env="test",
             interactive=False,
-            drop=False,
+            drop=True,
```

Comment:
Do we then always test with drop=True or do we also run the tests with drop=False? A bit hard to say without going outside the code review.

Reply:
Drop is behind a if statement, so I do not think there is much of a value to test with drop=False for this type of test. It was a mistake by me earlier to not set it to true.